### ISSUE TITLE
Help collect information about who would like to migrate from V2-V3

### ISSUE THREAD

**Issue by rowasc** [ushahidi/platform#3801]
## Overview
We are working on a v2-v3 migration tool. Initially we will do a CLI to evaluate usage and needs, we would like to be able to collect emails/info on deployers who are interested in the service to reach out to them. 

## Who is it for? Who are the users?
- Deployers who own a v2 deployment and may be interested in the migration service.

## What privacy impacts does it have?
- Does it change the personal data we collect? -> Yes. It saves an email (maybe other data to be determined but would like to minimize PII)
- Does it expose data that was previously private? - NO
- Does it send data to new services? or change how we process it? - Unclear, probably not? Need to spec more
- If privacy impacts are high risk, consider completing an impact assessment per GDPR (https://ico.org.uk/for-organisations/guide-to-the-general-data-protection-regulation-gdpr/accountability-and-governance/data-protection-impact-assessments/) - No, the privacy impact is not high risk.

### Who will this be deployed to?
Everyone.

### What documentation needs to change?
Maybe add some info in dotcom about this. 
We already published the roadmap item here https://docs.ushahidi.com/platform-developer-documentation/roadmap/v2-v3+-migration-tool

## Test script
TBD
### Who will test this?
Walter
### Where will this be deployed and tested?
DotCom?
## Deployment

*Does this deploy new services?* - A form
*Do the new services have health checks?* - No? Maybe.
*Can this be feature flagged and deployed disabled?* - No, it is not deployment specific

## Dependencies
Dotcom

Aha! Link: https://ushahiditeam.aha.io/features/PROD-167

**Comment by Erioldoesdesign** [comment]
@Angamanga Can we chat about this?

Essentially it's a page in dot com that would have text/copy about the v2-v3 migration project to gather/collect any people that we aren't actively reaching out to ourselves with  a simple form to collect:

- Deployment URL
- Email contact
- Why you want to migrate from v2-v3
- Best time to contact you/timezone
- Other info


There's a comms piece that goes around this so probably from design side:

- Emails
- Social posts

And possible some active prospecting to collect v2 users?

I'm aiming to do a mock up within the next few weeks but travelling intensity is imminent!

**Comment by Angamanga** [comment]
Sorry @Erioldoesdesign I missed this. It sounds alright and pretty straight forward :)

**Comment by Angamanga** [comment]
@rowasc @Erioldoesdesign @tuxpiper  I have started the form and connected it to a specific wufoo-form. Its in staging here: http://darwin.ushahidi.com/v3migration
1. What email-addresses do we want it to go to? 
2. Does this capture everything we want to know?
3. The copy... Some additional info in the right-hand column is needed

**Comment by Erioldoesdesign** [comment]
Shame on me for checking this 18 days later...

Anyways we talked about this on slack but I made edits in browser inspect which I'll upload here:

[Mirgration Page edits.zip](https://github.com/ushahidi/platform/files/3963484/Mirgration.Page.edits.zip)

But it's a zip!

Email address wise...it essentially needs to be customer support related communication. Wherever it goes there needs to be an auto-thank message and the ability for the core team to see these entries.

Re capturing everything - yes for sure I don't think it needs to capture anything else but we should be sure that email and deployment URL are mandatory for the form to submit.
